There was an error reported in the workflow but the test passed: https://github.com/NeurodataWithoutBorders/nwbinspector/actions/runs/8951656442/job/24588206055?pr=459#step:6:120

```
Inspecting NWBFiles...: 100%|██████████| 1/1 [00:00<00:00,  2.61it/s]
                                                                     
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.9/x64/bin/nwbinspector", line 8, in <module>
    sys.exit(inspect_all_cli())
             ^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nwbinspector/nwbinspector.py", line 2[93](https://github.com/NeurodataWithoutBorders/nwbinspector/actions/runs/8951656442/job/24588206055?pr=459#step:6:94), in inspect_all_cli
    json.dump(obj=json_report, fp=fp, cls=InspectorOutputJSONEncoder)
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/json/__init__.py", line 179, in dump
    for chunk in iterable:
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/json/encoder.py", line 432, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/json/encoder.py", line 406, in _iterencode_dict
    yield from chunks
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/json/encoder.py", line 406, in _iterencode_dict
    yield from chunks
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/json/encoder.py", line 439, in _iterencode
    o = _default(o)
        ^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nwbinspector/nwbinspector.py", line 56, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/json/encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Version is not JSON serializable
tests/test_inspector.py::TestInspector::test_command_line_runs_saves_json PASSED
```
Is that expected?

--- a/src/nwbinspector/nwbinspector.py
+++ b/src/nwbinspector/nwbinspector.py
@@ -52,6 +52,8 @@ class InspectorOutputJSONEncoder(json.JSONEncoder):
             return o.__dict__
         if isinstance(o, Enum):
             return o.name
+        if isinstance(o, Version):
+            return str(o)
         else:
             return super().default(o)
 

